unittests/files/opt/seagate/sspl/setup/resource_map/test_server.py

```diff
@@ -102,6 +102,52 @@ def test_build_platform_sensor_data_current(self, sdr_type):
         resp = self.server_map.get_platform_sensors_info()
         assert resp["Current"] == []
 
+    @patch("framework.platforms.server.sas.SAS.get_port_data")
+    @patch("framework.platforms.server.sas.SAS.get_port_list")
+    @patch("framework.platforms.server.sas.SAS.get_host_list")
+    def test_get_sas_hba_info(self, host_list, port_list, port_data):
+        host_list.return_value = ['host1']
+        port_list.return_value = ['port-1:0']
+        port_data.return_value = {
+            "port_id": "sas_port-1:0",
+            "state": "running",
+            "sas_address": "0x500c0fff0a98b000"
+        }
+        resp = self.server_map.get_sas_hba_info()
+        assert resp[0]['uid'] == "SASHBA-1"
```

Comment:
how always these values can be expected, it really depends on what system detects right.

Reply:
In Unit test, since sspl might not be deployed, we are patching the function calls made by `get_sas_hba_info()` with the responses specified on line `109-119`. So, these are hardcoded values, we are kind of checking only if the function is processing the responses correctly and putting them in the right schema or not